docs/03-extensions/adyen/reference/environment-variables.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Environment variables"
+description:
+  "This reference lists environment variables used by the Adyen extension"
+sidebar_position: 1
+---
+
+<p>{frontMatter.description}</p>
+
+## Debug
+
+This extension will output some debug informations if Front-Commerce is run with
+the `DEBUG` environment variable containing `front-commerce:payment:adyen`.
+
+## Required environment variables
```

Comment:
**issue:** these variables aren't required every time. Only in some flavors.

**suggestion:** split this section with variables per flavor